src/CommunityToolkit.Maui.Core/Interfaces/IDrawingView.shared.cs

```diff
@@ -53,6 +53,23 @@ public interface IDrawingView : IView
 	/// </summary>
 	void Clear();
 
+	/// <summary>
+	/// Event occurred when drawing started
+	/// </summary>
+	/// <param name="point">Last drawing point</param>
+	void DrawingStarted(PointF point);
+	
+	/// <summary>
+	/// Event occurred when drawing cancelled
+	/// </summary>
+	void DrawingCancelled();
+	
+	/// <summary>
+	/// Event occurred when drawing
+	/// </summary>
+	/// <param name="point">Last drawing point</param>
+	void OnDrawing(PointF point);
```

Comment:
Small nit - let's rename these two slightly to be consistent with `OnDrawing`. 

```suggestion
	/// <summary>
	/// Event occurred when drawing started
	/// </summary>
	/// <param name="point">Last drawing point</param>
	void OnDrawingStarted(PointF point);
	
	/// <summary>
	/// Event occurred when drawing cancelled
	/// </summary>
	void OnDrawingCancelled();
```

Conventionally, `DrawingStarted` and `DrawingCancelled` would be the name of the events, and `OnDrawingStarted` and `OnDrawingCancelled` would be the methods executed when the events fire.

Reply:
should we rename OnDrawingLineCompleted in that case as well?

Reply:
Dang - good catch! We must've missed that in the original PR.

Changing the APO would be a breaking change. 

In a perfect world, I agree that we should do it in this PR. 

I'm OK with leaving it named `DrawingLineCompleted` for now, but I'll defer the decision to you. 

I see two paths forward:
1. Change `DrawingLineCompleted` -> `OnDrawingLineCompleted`, update the docs and add the https://github.com/CommunityToolkit/Maui/labels/breaking%20change label
2. Open a Proposal and change it in a future PR that we'll merge alongside https://github.com/CommunityToolkit/Maui/pull/1367